ui/apps/platform/src/Containers/NetworkGraph/deployment/DeploymentFlows.tsx

```diff
@@ -169,6 +178,18 @@ function DeploymentFlow() {
     return (
         <div className="pf-u-h-100 pf-u-p-md">
             <Stack hasGutter>
+                <StackItem>
+                    <Flex>
+                        <FlexItem flex={{ default: 'flex_1' }} />
```

Comment:
What does this empty flex item do?

Reply:
It's just a placeholder for the eventual search input that I'll add soon